lib/ansible/modules/cloud/amazon/ec2_vpc_endpoint.py

```diff
@@ -366,16 +359,14 @@ def main():
                 module.fail_json(msg="Error - no region provided")
         else:
             module.fail_json(msg="Can't retrieve connection information - " + str(e),
-                             exception=traceback.format_exc(),
-                             **camel_dict_to_snake_dict(e.response))
+                             exception=traceback.format_exc())
```

Comment:
I believe you can remove this entire try/except. Having boto uninstalled should not cause NameError failure since https://github.com/ansible/ansible/commit/de5f8f1ec4ae66970cf13af8120d66dd619a782c